src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -83,6 +83,7 @@
     <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
       <Link>Common\System\PlatformDetection.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)'=='netstandard1.7'" Include="System\Math.1.7.cs" />
```

Comment:
If they are uniquely split then I wouldn't include anything unique in the name. However the file name conflicts and this is unique netstandard17 then I would name it Math.netstandard17.cs. The name should match the exact TargetGroup. I would also be fine putting them in a folder with that name.